bundle/manifests/volsync.clusterserviceversion.yaml

```diff
@@ -402,4 +412,11 @@ spec:
   minKubeVersion: 1.20.0
   provider:
     name: Red Hat
+  relatedImages:
```

Comment:
Do these get inserted into the csv in the `make bundle` step somehow now?

Nothing wrong with them but we may need to confirm later that the downstream build process updates this list of relatedImages rather than adding to it.  For example in the latest downstream CSV they look like this:

```
relatedImages:
  - name: volsync-rhel8-895405784c9ab0a2ed261e3a64559177626b307b1d79f27bea17ffa859345c8d-annotation
    image: registry.redhat.io/rhacm2/volsync-rhel8@sha256:895405784c9ab0a2ed261e3a64559177626b307b1d79f27bea17ffa859345c8d
  - name: kube-rbac-proxy
    image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:27540b17bf6e76916c60a650fa15f0ef034c2b4f8dfe2832c1a9da57368b744c
  - name: manager
    image: registry.redhat.io/rhacm2/volsync-rhel8@sha256:895405784c9ab0a2ed261e3a64559177626b307b1d79f27bea17ffa859345c8d
  - name: rsync_container
    image: registry.redhat.io/rhacm2/volsync-mover-rsync-rhel8@sha256:42a3ff104b43c0b33e136a4d1f0cd8a762324747a8379bd1a587fffd198b2b71
  - name: rclone_container
    image: registry.redhat.io/rhacm2/volsync-mover-rclone-rhel8@sha256:f7b1552d6531ccc2375b406e8d9da0d54f678a561fe12c8943ad26204f5326cf
  - name: restic_container
    image: registry.redhat.io/rhacm2/volsync-mover-restic-rhel8@sha256:53de358ba27c6b6cff2e6d80cbe937e540e72e32fb6e31747975dbfedca0a1dd
```

Just want to make sure we don't end up with a `rsync_container` related image and another one called `rsync-container` that still refers to quay.io.  Honestly I don't think they're even used as the env vars point to the exact image locations rather than using the values from relatedImages.

Reply:
I think it's a new (or fixed) thing in the sdk generate bundle command... I didn't add them by hand.
There was a fix to the sdk where it was missing some image references, but that's all I see in the release notes.

Reply:
Actually, it looks like it was developed as a feature in January. Since we're upgrading from 1.15 (Nov-ish), this is new.